frontend/src/pages/WorkspaceTeam/components/AllMembers.tsx

```diff
@@ -185,7 +187,7 @@ export const RoleOptions: Option[] = [
 	},
 ]
 
-const GRID_COLUMNS = ['3fr', '3fr', 'auto', '1fr', '30px']
+const GRID_COLUMNS = ['3fr', '3fr', 'auto', '1fr', '2fr', '30px']
```

Comment:
Not blocking, but not sure if this should be a fixed pixel width since the date formats are pretty constant? Looks like the columns can get a little shifted.